src/ert/_c_wrappers/enkf/queue_config.py

```diff
@@ -19,7 +19,7 @@ class QueueConfig:
     )
 
     @classmethod
-    def from_dict(cls, config_dict) -> QueueConfig:
+    def from_dict(cls, config_dict, collected_errors: List[ErrorInfo]) -> QueueConfig:
```

Comment:
The interface for this function becomes very difficult to follow, except in the context of `ErtConfig.from_file`. I suggest that it should raise ConfigValidationError that `ErtConfig.from_file` catches and then appends to its list of collected errors.